src/encoded/authentication.py

```diff
@@ -192,19 +210,22 @@ def get_user_info(request):
         request.set_property(get_user_info, "user_info", True)
         return email
 
-    def get_token_info(self, token, request):
+    @staticmethod
+    def get_token_info(token, request):
         '''
```

Comment:
Very minor: If you see any triple-single-quotes in these files and want to upgrade them triple-double-quotes, feel free. PEP8 standard is triple-double-quotes for doc strings.